### DISCUSSION TITLE
How do I resolve pr conflict

### DISCUSSION THREAD

**Discussion by ExChuckNorris** [community/community#122665] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Please give me an explanation that is easy for a beginner to understand.

**Comment by gumob** [comment, marked as answer]
Read the documentation carefully before asking questions.
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github
